### DISCUSSION TITLE
Brew install problem on clean format Big Sur

### DISCUSSION THREAD

**Discussion by SteffRainville** [Homebrew/discussions#146] [Everyday usage]
Is the following issue of concern or a blocker to use Brew

Receiving objects: 100% (834187/834187), 326.76 MiB | 7.69 MiB/s, done.
Resolving deltas: 100% (561752/561752), done.
Tapped 2 commands and 5344 formulae (5,629 files, 359MB).
Already up-to-date.
Error: Failed to link all completions, docs and manpages:
  Permission denied @ rb_file_s_symlink - (../../../Homebrew/manpages/README.md, /usr/local/share/man/man1/README.md)
Failed during: /usr/local/bin/brew update --force

**Comment by SMillerDev** [comment]
Homebrew isn't supported on Big Sur yet, however this fix is easy.

You don't have access to `/usr/local/share/man/man1/README.md` you should fix your permissions.

**Comment by NOTiFYcouk** [comment]
Simplest solution that worked when upgrading to Big Sur. A clean install may not require the solution below. I believe there is some inconsistency with the way CLT is upgraded.

sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install
